docs/components/early-access/overview.md

```diff
@@ -0,0 +1,40 @@
+---
+id: overview
+title: Overview
+sidebar_label: Overview
+---
+
+Introducing early access - a space to explore new features and components currently in development by Camunda.
+
+Early access includes two categories of features:
+
+## Experimental features
+
+Experimental features are very early-stage ideas and functionalities under exploration. This allows Camunda to innovate quickly and test new concepts.
+
+You can test out these features and provide additional feedback to help Camunda understand how they perform in real-world scenarios.
+
+:::note
+These features may be unstable, and subject to significant changes or removal.
+:::
+
+## Alpha features
+
+Alpha features are more developed and closer to becoming part of the product but may not yet be fully optimized or supported. Selected Camunda features and components are released as alpha versions to provide early access. By testing these features, you have the opportunity to participate in their development by sharing feedback before they reach General Availability (GA).
```

Comment:
Do we want to link to the Alpha features page? https://docs.camunda.io/docs/next/reference/alpha-features/

Reply:
Honestly we could probably move this entire page over as an "overview" to the alpha features section, but I consider that out of scope for this PR. Will open an additional issue to revisit.

Reply:
I was going to provide similar feedback - we include definitions or explanations for alphas and minors on the [release policy page](https://docs.camunda.io/docs/next/reference/release-policy/#alpha-features-and-releases) as well. These should be linked so they reference each other. I'm ok with this being handled in a separate PR.

Reply:
Cool. I'll launch another PR. Can I get a final approval from one of you so I can go ahead and release this?